### PR TITLE
Add pretrain vectors

### PR DESCRIPTION
**Pull request by nsorros**
Adds this new feature of SpaCy https://spacy.io/usage/v2-1#pretraining.

The idea of the feature is to pretrain some vectors in an largest dataset with no labels and use those vectors in the downstream task. It should have the same effect as BERT like transfer learning.

### CODE REVIEW

File: wellcomeml/ml/spacy_classifier.py [hunk 1]
```diff
@@ -112,6 +114,11 @@ def yield_train_data(X_train, Y_train):
             optimizer = self.nlp.begin_training()
             optimizer.alpha = self.learning_rate
             #optimizer.L2 = 1e-4
+
+            if self.pre_trained_vectors:
```
Comment (lizgzil) [inline wellcomeml/ml/spacy_classifier.py]: might be good to name `pre_trained_vectors` so it's clear it is a path type?

File: wellcomeml/ml/spacy_classifier.py [hunk 2]
```diff
@@ -112,6 +114,11 @@ def yield_train_data(X_train, Y_train):
...
+                with open(self.pre_trained_vectors, "rb") as f:
+                    self.textcat.model.tok2vec.from_bytes(f.read())
```
Comment (lizgzil) [inline wellcomeml/ml/spacy_classifier.py]: where does `textcat.model.tok2vec` get initialised if `pre_trained_vectors=None`?

File: wellcomeml/ml/spacy_classifier.py (hunk 1 above)
Comment (nsorros) [reply wellcomeml/ml/spacy_classifier.py]: what is the name suggestion? `pre_trained_vectors_path`?

File: wellcomeml/ml/spacy_classifier.py (hunk 2 above)
Comment (nsorros) [reply wellcomeml/ml/spacy_classifier.py]: inside spacy

File: wellcomeml/ml/spacy_classifier.py (hunk 1 above)
Comment (lizgzil) [reply wellcomeml/ml/spacy_classifier.py]: sure!

**Comment by aCampello** [comment]
LGTM too - I re-ran the pipeline and it passes now.
